django/db/migrations/serializer.py

```diff
@@ -270,62 +271,64 @@ def serialize(self):
         return "uuid.%s" % repr(self.value), {"import uuid"}
 
 
+def is_deconstructs(value):
+    unsuitable_types = (models.Field, models.manager.BaseManager, Operation, type)
```

Comment:
Maybe it would be simpler not to include these types in the registration process and leave them in the if-chain before `DeconstructableSerializer` so this special casing isn't required. It seems to make things more difficult to understand.